src/components/UI/OrganizationList/OrganizationList.js

```diff
@@ -19,13 +19,11 @@ const OrganizationList = ({ provider, ...props }) => {
           <StyledTableHeader centered={true}>Clusters</StyledTableHeader>
           <StyledTableHeader centered={true}>Members</StyledTableHeader>
 
-          {provider !== Providers.KVM && (
+          {(provider === Providers.AWS || provider === Providers.AZURE) && (
```

Comment:
This piece of business logic is duplicated in `src/components/Organizations/Detail/View.js`

and `src/components/UI/OrganizationList/OrganizationListRow.js`

I realise thats not something you introduced. But if you feel like extracting it to a common place, that would be 💯

Reply:
Very good idea 👍🏻